src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceAfterCheck.java

```diff
@@ -136,12 +138,15 @@ public void setAllowLineBreaks(boolean allowLineBreaks) {
     public void visitToken(DetailAST ast) {
         final DetailAST whitespaceFollowedAst = getWhitespaceFollowedNode(ast);
 
-        final int whitespaceColumnNo = getPositionAfter(whitespaceFollowedAst);
-        final int whitespaceLineNo = whitespaceFollowedAst.getLineNo();
+        if (whitespaceFollowedAst.getNextSibling() == null
+                || whitespaceFollowedAst.getNextSibling().getType() != TokenTypes.ANNOTATIONS) {
+            final int whitespaceColumnNo = getPositionAfter(whitespaceFollowedAst);
```

Comment:
Whitespace package is not yet enforced pitest coverge in CI , please generate report and place screenshot that you covered all your changes

Reply:
[Pitest report](https://sergileon.github.io/3300/PITEST/index.html)